student/src/main/java/tds/student/web/handlers/TestResponseHandler.java

```diff
@@ -141,8 +166,17 @@ private void updateResponses (HttpServletRequest request, HttpServletResponse re
       // If scoring is done sychronously, then score and then updateDB
       // If scoring is done asynchronously, then updateDB (with score -1 and
       // status WaitingForMachineScore) and then score
+      _eventLogger.info(((EventInfo)request.getAttribute(LoggingInterceptor.EVENT_INFO))
+        .withMessage("_itemScoringService.updateResponses")
+        .withCheckpoint(SERVICE_CALL.name()));
+
       List<ItemResponseUpdateStatus> responseResults = null;
         responseResults = _itemScoringService.updateResponses (testOpp.getOppInstance (), responseReader.getResponses (), responseReader.getPageDuration());
+
+      _eventLogger.info(((EventInfo)request.getAttribute(LoggingInterceptor.EVENT_INFO))
+        .withMessage("_itemScoringService.updateResponses")
+        .withCheckpoint(SERVICE_RETURN.name()));
```

Comment:
I hate to say it, but this still feels pretty invasive at the @Controller level... What I'd love is to just be able to call:
```
LOG.info("updating responses");
``
using the standard SLF4J logger down inside the ItemScoringService implementation and have it be bound to the overall request log set.